engineering-feedback/feedback-faq.md

```diff
@@ -0,0 +1,39 @@
+# Engineering Feedback Frequently Asked Questions (F.A.Q.)
+
+The questions below are common questions related to the feedback process. The answers are intended to help both Microsoft employees and customers.
+
+## When should I submit feedback versus creating an issue on GitHub, UserVoice, or sending an email directly to a Microsoft employee?
+
+It is appropriate to do both. As a customer or Microsoft employee, you are empowered to create an issue or submit feedback via the medium appropriate for service.
+
+In addition to an issue on GitHub, feedback on UserVoice, or a personal email, Microsoft employees in CSE should submit feedback via CSE Feedback.  In doing so, please reference the GitHub issue, UserVoice feedback, or email by including a link to the item or attaching the email.
```

Comment:
Maybe link the first mention of CSE to code-with-engineering-playbook/CSE.md just in case